oviewer/ovstyle.go

```diff
@@ -138,3 +140,18 @@ func applyStyle(style tcell.Style, s OVStyle) tcell.Style {
 
 	return style
 }
+
+// underLineStyle sets the underline style.
+// only support 0-5.
+// 0: None, 1: Single, 2: Double, 3: Curly, 4: Dotted, 5: Dashed
```

Comment:
About this

https://github.com/noborus/ov/pull/680#discussion_r1909965613

Maybe you could link the constants here with a godoc link

Something like this (pseudo code)

```suggestion
// underLineStyle sets the underline style.
// only support 0-5.
// 0: [tview.WhateverNone], 1: [tview.WhateverSingle], 2: Double, 3: Curly, 4: Dotted, 5: Dashed
```